tests/rendering/markdown-additional.md

```diff
@@ -63,7 +65,18 @@ This paragraph has a red background color.{style=background-color:red}
 > [!CAUTION]
 > Do not do this and that!
 
-With a custom title:
+### With a custom title
 
 > [!NOTE] Foo bar
-> Hello
+> 'Note' with a custom title
+
+### Using custom markers ([Obsidian Callout](https://help.obsidian.md/Editing+and+formatting/Callouts) style)
+
+> [!CUSTOM]
+> Something more special
+
+> [!fOoBaR]
+> The marker is case-insensitive and turns into Title Case
```

Comment:
Check if it actually is "Title Case",
However since multi-word markers don't currently work in the plugin, it's not super relevant at the moment

Reply:
Confirmed that in Obsidian it's like this:
```md
> [!hello world]
```
gives **Hello world** so first letter capitalized only